scripts/hca/ingest-hca-collection.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
 
-import json, os
-from urllib.request import Request, urlopen
+import json
+import os
+
+import requests
 from tqdm import tqdm
 
 upsert_url = "https://catalog.dsde-dev.broadinstitute.org/api/v1/datasets"
-policy_url = "https://jade.datarepo-dev.broadinstitute.org/api/repository/v1/snapshots/{id}/policies/steward/members"
+policy_url = (
```

Comment:
Why does this one need to be paren'd and the one above isnt? Is it a line length thing?

Reply:
Sorry, I meant to revert the changes to this file. These changes are both some improvements and changes from updating to conform to PEP8 (which has shorter lines, among other things). Black is different than PEP8 so these changes aren't necessary for this PR.